cfme/containers/container.py

```diff
@@ -58,3 +58,13 @@ def get_detail(self, *ident):
         """
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
+
+    @staticmethod
+    def get_container_names():
+        sel.force_navigate('containers_containers')
+        return map(lambda r: r.name.text, list_tbl.rows())
```

Comment:
Also.. it doesn't make much sense to have `container` in the method name because then you will call `Container.get_container_names` which is redundant - it would suffice to say `Container.get_names`, or rather `Container.get_all_names`, because the context is clear from the class' name.